packages/plugin-ext/src/api/rpc-protocol.ts

```diff
@@ -387,6 +388,15 @@ namespace ObjectsTransferrer {
                 $type: SerializedObjectType.VSCODE_URI,
                 data: uri.toString()
             } as SerializedObject;
+        } else if (value && typeof value === 'object') {
+            if (cache.indexOf(value) !== -1) {
+                try {
+                    return JSON.parse(JSON.stringify(value));
+                } catch (e) {
+                    return;
```

Comment:
VS Code translates circular ref to `null`

Reply:
changed to `return undefined`